2_process/src/ModelPlotting_ms.R

```diff
@@ -4,52 +4,50 @@
 # Date:                  #
```

Comment:
The template about writer and date can be removed, as well. We rely on git histories to provide this information.